build.gradle

```
@@ -1,3 +1,10 @@
+plugins {
+    id "com.automattic.android.fetchstyle"
+    id "io.gitlab.arturbosch.detekt"
+    id "androidx.navigation.safeargs.kotlin" apply false
+    id "com.android.library" apply false
```

There's `Warning: Please apply google-services plugin at the bottom of the build file.` warning in build log, the same as you suggested in Woo PR (https://github.com/woocommerce/woocommerce-android/pull/5414#discussion_r764338660) I'm suggesting the same 🙂:

```suggestion
    id "com.android.library" apply false
    id 'com.google.gms.google-services' apply false
```

---

Addressed in e3c97b8acf216a6c9ea47b4eda3a3d1d45ca3ac1.